internal/validations/clusterlogforwarder/validate_clusterlogforwarder_service_account.go

```diff
@@ -130,6 +130,13 @@ func gatherPipelineInputs(clf loggingv1.ClusterLogForwarder) (sets.String, bool)
 				if hasInfraNamespaces(input.Application.Namespaces) {
 					inputTypes.Insert(loggingv1.InputNameInfrastructure)
 				}
+				if len(input.Application.Includes) > 0 {
+					for _, in := range input.Application.Includes {
+						if infraNamespaces.MatchString(in.Namespace) {
```

Comment:
Isn't there a need to check if this NS is an infra namespace?  This is the companion check to line 130 is it not?

Reply:
line135 checks for infra?    Not sure I understand the question.

Reply:
:+1: I missed the fact "infraNamespaces" is a RE